server/api/expense-pool/expense/index.js

```diff
@@ -1,23 +1,23 @@
+/**
+ * POST  /api/expense-pools/:id/expenses            -> create
+ * GET   /api/expense-pools/:id/expenses            -> all expenses for a pool
+ * GET   /api/expense-pools/:id/expenses/:expenseId -> get
+ * PUT   /api/expense-pools/:id/expenses/:expenseId -> update
+ * PATCH /api/expense-pools/:id/expenses/:expenseId -> update
+ */
+
 import express from 'express';
 import controller from './expense.controller';
 
-const router = express.Router({ mergeParams: true });
```

Comment:
Why are we getting rid of mergeParams? Won't this make us loose access to multiple id's in the route?